src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/VisitNotificationEventService.kt

```diff
@@ -193,4 +195,61 @@ class VisitNotificationEventService(
   fun getNotificationCount(): Int {
     return this.visitNotificationEventRepository.getNotificationGroups() ?: 0
   }
+
+  fun getPrisonerVisitNotifications(): List<NotificationGroupDto> {
+    val now = LocalDate.now()
+
+    val list = mutableListOf<NotificationGroupDto>()
+    list.add(
```

Comment:
monir comment - can be ignored - we could add a TODO / comment to say this is a temporary response.